docs-src/references/web-ui-configuration.md

```diff
@@ -158,6 +158,49 @@ codec:
   passAccessToken: false
 ```
 
-# disableWriteActions
+## disableWriteActions
 
-Prevents the user from terminating or canceling Workflow Executions from the Web UI.
+Prevents the user from executing Workflow Actions on the Web UI.
+
+This option affects Bulk Actions for Recent Workflows as well as Workflow Actions on the Workflow Details page.
+
+```yaml
+disableWriteActions: false
+```
+
+:::note
+`disableWriteActions` overrides the configuration values of each individual Workflow Action.
+Setting this variable to `true` will disable all Workflow Actions on the Web UI.
+:::
+
+## workflowTerminatedDisabled
+
+Prevents the user from terminating Workflow Executions from the Web UI.
+
+```yaml
+workflowTerminatedDisabled: false
+```
+
+## workflowCancelDisabled
+
+Prevents the user from canceling Workflow Executions from the Web UI.
+
+```yaml
+workflowCancelDisabled: false
+```
+
+## workflowSignalDisabled
+
+When enabled, prevents the Workflow from sending or receiving Signals from the Web UI.
```

Comment:
I think we can just change this to: `Prevents the user from Signaling workflows from the Web UI.` to keep it more similar to the others.